syft/execution/plan.py

```diff
@@ -274,6 +275,20 @@ def replace_with_placeholders(self, obj, arg_ids, result_ids, **kw):
             else:
                 return None
 
+    def replace_ids_with_placeholders(self, obj):
+        """
+        Replace in an object all ids with Placeholders
```

Comment:
This only applies to `ObjectIds`/`PlaceholderIds` now, right?

Reply:
Yep! All objects in Plan actions should be placeholders (except constants) and they are all encoded with an ObjectId